daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actor-howto.md

```diff
@@ -1,9 +1,16 @@
-# Getting started with Actors
-
-## Prerequistes
-* [.Net Core SDK 3.1 or newer](https://dotnet.microsoft.com/download)
-* [Dapr CLI](https://github.com/dapr/cli)
-* [Dapr DotNet SDK](https://github.com/dapr/dotnet-sdk)
+---
+type: docs
+title: "Getting started with the Dapr actor .NET SDK"
+linkTitle: "Example"
+weight: 100000
+description: Try out .NET virtual actors with this example
+---
+
+## Prerequisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [.NET 3.1+](https://dotnet.microsoft.com/download) installed
```

Comment:
This is confusing, but needs to be clarified. The right branding for 3.1 is `.NET Core 3.1` - the right branding for 5.0 is `.NET 5+`

```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```